app/celery/process_delivery_status_result_tasks.py

```diff
@@ -34,222 +33,57 @@
 )
 @statsd(namespace='tasks')
 def process_delivery_status(
-    self,
+    self: Task,
     event: CeleryEvent,
-) -> bool:
+) -> None:
     """
-    This is a Celery task for updating the delivery status of a notification.
+    This is a Celery task for updating the delivery status of a Twilio notification.
```

Comment:
should this be more explicitly named that it processed only Twilio delivery statuses?